### PR TITLE
feat(LABY.net): add more detail and buttons, fix issues and logical order

### PR DESCRIPTION
**Pull request by Northernside**
**Describe the changes in this pull request:**
```diff
+ /badges is now displaying an own presence detail (Screenshot 1)
+ /cloak is now displaying an own presence detail (Screenshot 2)
+ Added buttons for certain links (Screenshot 3 - 8)

*** Fixed an issue where PreMiD can't display the viewed username
*** Fixed an issue where the LABY.net Presence freezes when viewing/searching for an invalid username/UUID
*** Changed some weird unlogical code (lmao)
```

<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

image
image
image
image
image
image

</details>

### CODE REVIEW

File: websites/L/LABY.net/presence.ts [hunk 1]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
-  } else if (document.location.pathname.startsWith("/settings"))
-    presenceData.details = "Viewing profile settings";
-  else if (document.location.pathname.startsWith("/server")) {
+    presenceData.state =
+      document.querySelector("div.col-12>h1")?.textContent ??
+      "an invalid account";
+    presenceData.buttons = [
+      {
+        label: `View ${
+          document.querySelector("div.col-12>h1")?.textContent
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:30]:
```suggestion
          document.querySelector("div.col-12>h1")?.textContent ?? "Unknown"
```

File: websites/L/LABY.net/presence.ts [hunk 2]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
+    presenceData.details = "Viewing a Minecraft skin";
+    presenceData.buttons = [
+      {
+        label: "View this Minecraft skin",
+        url: window.location.href
+      }
+    ];
+  } else if (document.location.pathname.startsWith("/cape")) {
+    presenceData.details = "Viewing cape:";
+    presenceData.state = document.querySelector("div.mb-1>h1").textContent;
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:45]:
```suggestion
    presenceData.state = document.querySelector("div.mb-1>h1")?.textContent ?? "Unknown";
```

File: websites/L/LABY.net/presence.ts [hunk 3]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
+    presenceData.buttons = [
+      {
+        label: `View ${
+          document.querySelector("div.mb-1>h1")?.textContent
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:49]:
```suggestion
          document.querySelector("div.mb-1>h1")?.textContent ?? "Unknown"
```

File: websites/L/LABY.net/presence.ts [hunk 4]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
+    presenceData.details = "Viewing a LabyMod cloak";
+    presenceData.buttons = [
+      {
+        label: "View this cloak",
+        url: window.location.href
+      }
+    ];
+  } else if (document.location.pathname.startsWith("/badge")) {
+    presenceData.details = "Viewing badge:";
+    presenceData.state = document.querySelector("div.mb-1>h1").textContent;
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:64]:
```suggestion
    presenceData.state = document.querySelector("div.mb-1>h1")?.textContent ?? "Unknown Badge";
```

File: websites/L/LABY.net/presence.ts [hunk 5]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
+    presenceData.buttons = [
+      {
+        label: `View ${
+          document.querySelector("div.mb-1>h1")?.textContent
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:68]:
```suggestion
          document.querySelector("div.mb-1>h1")?.textContent ?? "Unknown Badge"
```

File: websites/L/LABY.net/presence.ts [hunk 6]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
-  } else if (document.location.pathname.startsWith("/badge")) {
-    presenceData.details = "Viewing badge:";
-    presenceData.state = document.querySelector("div.mb-1>h1").textContent;
-  } else if (document.location.pathname.startsWith("/cape")) {
-    presenceData.details = "Viewing cape:";
-    presenceData.state = document.querySelector("div.mb-1>h1").textContent;
+    presenceData.buttons = [
+      {
+        label: `View ${
+          document.querySelector("div.server-info-wrapper>h1")?.textContent
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:81]:
```suggestion
          document.querySelector("div.server-info-wrapper>h1")?.textContent ?? "Unknown Cape"
```

File: websites/L/LABY.net/presence.ts [hunk 7]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
+          document.querySelector("div.mb-1>h1")?.textContent
+        } cape`,
+        url: window.location.href
+      }
+    ];
+  } else if (document.location.pathname.startsWith("/cloak")) {
+    presenceData.details = "Viewing a LabyMod cloak";
+    presenceData.buttons = [
+      {
+        label: "View this cloak",
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:58]:
```suggestion
        label: "View this Cloak",
```

File: websites/L/LABY.net/presence.ts [hunk 8]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
+    presenceData.buttons = [
+      {
+        label: `View ${
+          document.querySelector("div.mb-1>h1")?.textContent
+        } cape`,
+        url: window.location.href
+      }
+    ];
+  } else if (document.location.pathname.startsWith("/cloak")) {
+    presenceData.details = "Viewing a LabyMod cloak";
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:55]:
```suggestion
    presenceData.details = "Viewing a LabyMod Cloak";
```

File: websites/L/LABY.net/presence.ts [hunk 9]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
+  } else if (document.location.pathname.startsWith("/skin")) {
+    presenceData.details = "Viewing a Minecraft skin";
+    presenceData.buttons = [
+      {
+        label: "View this Minecraft skin",
+        url: window.location.href
+      }
+    ];
+  } else if (document.location.pathname.startsWith("/cape")) {
+    presenceData.details = "Viewing cape:";
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:44]:
```suggestion
    presenceData.details = "Viewing Cape:";
```

File: websites/L/LABY.net/presence.ts [hunk 10]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
   else if (document.location.pathname.startsWith("/capes"))
     presenceData.details = "Viewing Minecraft capes";
+  else if (document.location.pathname.startsWith("/cloaks"))
+    presenceData.details = "Viewing LabyMod cloaks";
+  else if (document.location.pathname.startsWith("/badges"))
+    presenceData.details = "Viewing LABY.net badges";
+  else if (document.location.pathname.startsWith("/settings"))
+    presenceData.details = "Viewing profile settings";
   else if (document.location.pathname.startsWith("/@")) {
     presenceData.details = "Viewing profile of:";
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:23]:
```suggestion
    presenceData.details = "Viewing Profile of:";
```

File: websites/L/LABY.net/presence.ts [hunk 11]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
   };
   if (document.location.pathname.startsWith("/skins"))
     presenceData.details = "Viewing skins";
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:13]:
```suggestion
    presenceData.details = "Viewing Skins";
```

File: websites/L/LABY.net/presence.ts [hunk 12]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
-  else if (document.location.pathname.startsWith("/skin"))
-    presenceData.details = "Viewing skins";
-  else if (document.location.pathname.startsWith("/cloaks"))
-    presenceData.details = "Viewing LabyMod cloaks";
   else if (document.location.pathname.startsWith("/capes"))
     presenceData.details = "Viewing Minecraft capes";
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:15]:
```suggestion
    presenceData.details = "Viewing Minecraft Capes";
```

File: websites/L/LABY.net/presence.ts [hunk 13]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
+  else if (document.location.pathname.startsWith("/cloaks"))
+    presenceData.details = "Viewing LabyMod cloaks";
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:17]:
```suggestion
    presenceData.details = "Viewing LabyMod Cloaks";
```

File: websites/L/LABY.net/presence.ts [hunk 14]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
+  else if (document.location.pathname.startsWith("/badges"))
+    presenceData.details = "Viewing LABY.net badges";
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:19]:
```suggestion
    presenceData.details = "Viewing LABY.net Badges";
```

File: websites/L/LABY.net/presence.ts [hunk 15]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
+  else if (document.location.pathname.startsWith("/settings"))
+    presenceData.details = "Viewing profile settings";
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:21]:
```suggestion
    presenceData.details = "Viewing Profile Settings";
```

File: websites/L/LABY.net/presence.ts [hunk 16]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
+    presenceData.buttons = [
+      {
+        label: `View ${
+          document.querySelector("div.col-12>h1")?.textContent
+        }'s profile`,
+        url: window.location.href
+      }
+    ];
+  } else if (document.location.pathname.startsWith("/skin")) {
+    presenceData.details = "Viewing a Minecraft skin";
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:36]:
```suggestion
    presenceData.details = "Viewing a Minecraft Skin";
```

File: websites/L/LABY.net/presence.ts [hunk 17]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
+    presenceData.buttons = [
+      {
+        label: "View this Minecraft skin",
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:39]:
```suggestion
        label: "View this Minecraft Skin",
```

File: websites/L/LABY.net/presence.ts [hunk 18]
```diff
@@ -11,30 +11,78 @@ presence.on("UpdateData", () => {
...
+    presenceData.buttons = [
+      {
+        label: `View ${
+          document.querySelector("div.mb-1>h1")?.textContent
+        } badge`,
+        url: window.location.href
+      }
+    ];
+  } else if (document.location.pathname.startsWith("/server")) {
     presenceData.details = "Viewing server:";
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:74]:
```suggestion
    presenceData.details = "Viewing Server:";
```

**Review by 7loud — COMMENTED** [review]
Some suggestions

**Comment by Northernside** [comment]
I'm rerunning Lint again tomorrow. Got no time today anymore

**Comment by Northernside** [comment]
Oh my god, me the idiot forgot to change branches. Imma try to undo this

**Comment by ImRodry** [comment]
This needs a rebase

**Comment by Northernside** [comment]
> This needs a rebase

Hey there,
I'm not that familiar with Git yet - so my question is: how do I rebase? Via the CLI, right? But how? I tried using [this](https://gist.github.com/nnja/a1888523ec851c6b94b2647956d5c3b4) tutorial but I don't know how many commits I need to rebase nor how to check for how many I need. Also if I do the alternate way with `git merge-base patch-2 master`, it says following: `fatal: Not a valid object name patch-2`. And yes, I CD'ed into the local repository directory.

**Review by Bas950 — APPROVED** [review]

File: websites/H/HBO GO/dist/metadata.json [hunk 19]
```diff
@@ -1,34 +1,32 @@
...
-  "tags": [
-    "video",
-    "media"
-  ]
+	"$schema": "https://schemas.premid.app/metadata/1.6",
+	"author": {
+		"name": "PreMiD",
+		"id": "503557087041683458"
+	},
+	"service": "HBO GO",
```
Comment (Bas950) [inline websites/H/HBO GO/dist/metadata.json:7]: undo changes to hbo file

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**